assets/js/map.js

```diff
@@ -0,0 +1,242 @@
+(function(exports) {
+
+  // our DOM container and status message
+  var map = d3.select("#team-map")
+        .classed("loading", true),
+      status = map.append("p")
+        .html("Loading the map...");
+
+  // names and geographic locations of airport codes
+  var locations = [
```

Comment:
Are there instructions for how someobody would add new locations? Could this be automated? I would imagine that if a member from a new city joins our team, we'll forget to add the new city.